### PR TITLE
sscep: init at 0.10.0

### PR DESCRIPTION
**Pull request by stigtsp**
###### Motivation for this change

Adds the `sscep` cli tool for the SCEP protocol

https://github.com/certnanny/sscep
https://repology.org/project/sscep/versions

###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by Flakebi — APPROVED** [review]
Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>sscep</li>
  </ul>
</details>

**Comment by c0bw3b** [comment]
@ofborg build sscep

(checking Darwin builds)

File: pkgs/tools/security/sscep/default.nix [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  buildInputs = [
+    openssl
+  ];
+
+  preConfigure = "./bootstrap.sh";
+
+  meta = with lib; {
+    description = "Command line client for the SCEP protocol";
+    homepage = "https://github.com/certnanny/sscep";
+    license = [ licenses.bsd3 licenses.openssl licenses.vsl10 ];
```
Comment (c0bw3b) [inline pkgs/tools/security/sscep/default.nix:39]: [From upstream notice](https://github.com/certnanny/sscep/blob/master/COPYING#L2-L25) it looks more like BSD-2.

And I'm not sure we need to list the OpenSSL license here, it's just linked.
I also can't find what part of the code is under the VSL-1.0 license. It's included in the COPYING notice but [their own spec file](https://github.com/certnanny/sscep/blob/master/scripts/sscep.spec#L10) does not mention it.

**Review by c0bw3b — COMMENTED** [review]
Builds and runs otherwise

Result of `nixpkgs-review pr 141630` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>sscep</li>
  </ul>
</details>

File: pkgs/tools/security/sscep/default.nix [hunk 2]
```diff
@@ -0,0 +1,42 @@
+{ lib, stdenv
```
Comment (onny) [inline pkgs/tools/security/sscep/default.nix:1]:
```suggestion
{ lib
, stdenv
```

File: pkgs/tools/security/sscep/default.nix (hunk 1 above)
Comment (onny) [inline pkgs/tools/security/sscep/default.nix:39]:
```suggestion
    license = with linceses; [ bsd3 openssl vsl10 ];
```

**Pull request closed without merging**